backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/transformer/DocumentationUnitTransformer.java

```diff
@@ -151,6 +151,8 @@ public static DocumentationUnitDTO transformToDTO(
       var managementData = updatedDomainObject.managementData();
 
       builder.scheduledPublicationDateTime(managementData.scheduledPublicationDateTime());
+      builder.lastPublicationDateTime(managementData.lastPublicationDateTime());
```

Comment:
We are not using this info, do we? I think lastPublicationDateTime is by the backend and we ignore what comes from the client?

Reply:
The `transformToDTO` is used in the `save` method of the repository before saving. The Scheduling Services saves the docUnit including setting the two dates and the email address.
https://github.com/digitalservicebund/ris-backend-service/blob/419a49d1683b8787e0cb052c8288462d350291c1/backend/src/main/java/de/bund/digitalservice/ris/caselaw/adapter/database/jpa/PostgresDocumentationUnitRepositoryImpl.java#L203-L203

However, especially with the email, we might want to have specific methods and a controller endpoint for our user case. Would be very interested in hearing your input on this. :-)